fbclient/client.py

```
@@ -154,11 +166,30 @@ def stop(self):
+        # Stop producers before consumers, and isolate every component so one
+        # extension failure cannot prevent the remaining resources from being
+        # released or escape into application shutdown code. No user callback
+        # is invoked while the client lifecycle lock is held.
+        for name, component in (
+                ('update processor', self._update_processor),
+                ('event processor', self._event_processor),
+                ('notice broadcaster', self._broadcaster),
+                ('data storage', self._data_storage)):
+            try:
+                component.stop()
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)